### DISCUSSION TITLE
Auto search/download the corresponding beatmap when importing a replay if the meatmap dose not exist

### DISCUSSION THREAD

**Discussion by LIII-Works** [ppy/osu#14076] [Ideas] [answered]
Sometimes when I try to import a replay file exported from stable to lazer, I dont have the corresponding beatmap, that specific one. I have to open my stable, look for beatmap page, download, import it into lazer. It is kinda inconvenient to do, especially for player who record or check replays frequently.

**Comment by peppy** [comment, marked as answer]
Duplicate of https://github.com/ppy/osu/discussions/14029. Please search first.
